themes/default/content/learn/abstraction-encapsulation/abstraction/index.md

```diff
@@ -1,12 +1,12 @@
 ---
+title_tag: Abstracting Pulumi Code | Learn Pulumi
 title: "Abstracting Pulumi Code"
 layout: topic
 date: 2021-11-17
 draft: false
 description: |
     Learn more about abstracting Pulumi code into classes, models, and objects.
-meta_desc: |
-    Learn more about abstracting Pulumi code into classes, models, and objects.
+meta_desc: Learn how to think in abstraction and how to abstract Pulumi code into reusable classes, models, and objects with this tutorial.
```

Comment:
what does it mean to "think in abstraction" i think we are gonna need to improve this a bit

Reply:
Yeah, fair callout. We'd taken it from one of the subheadings on that page (thinking in abstraction). 

How about something like: "In this tutorial, we'll cover the concept of abstraction and how to abstract Pulumi code into reusable classes, models, and objects."

Reply:
@tofuni your suggestion looks perfect, thank you!!